provider/haproxy/config/rsyslogd.conf

```diff
@@ -3,15 +3,12 @@ $UDPServerRun 8514
 $template CustomFormat,"%timegenerated% %HOSTNAME% %syslogtag%%msg%\n"
 $ActionFileDefaultTemplate CustomFormat
 
-/* info */
-if $programname contains 'haproxy' and $syslogseverity == 6 then
-    action(type="omfile" file="/var/log/haproxy/traffic")
-
-if $programname contains 'haproxy' and $syslogseverity-text == 'err' then
-    action(type="omfile" file="/var/log/haproxy/errors")
-
-/* notice */
-if $programname contains 'haproxy' and $syslogseverity <= 5 then
-    action(type="omfile" file="/var/log/haproxy/events")
-
-*.* /var/log/messages
+if $programname contains 'haproxy' then {
+  if $syslogseverity == 6 then
+      action(type="omfile" file="/var/log/haproxy/traffic")
+  if $syslogseverity-text == 'err' then
```

Comment:
@kinarashah events should have anything <= 5. I'd say error would have <=3. It is ok that they overlap, in case user wants to debug - debug and error events in the same log file ("events") makes it easier to figure out the events sequence. @cjellic your thoughts?